### PR TITLE
Fix HR issues.

### PR DESCRIPTION
**Pull request by haiodo**
# Contribution checklist

## Brief description

1. Allow to Show/Hide and edit existing attributes.
image

2. Fix Tracker Labels
3. Fix restore from backup
4. Fix class attribute bar mixins and default collapse/expand levels.
5. Fix control default properties display

Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

## Checklist

* [x] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/ui/src/components/calendar/DateRangePresenter.svelte [hunk 1]
```diff
@@ -318,7 +319,7 @@
     >
       {#if edits[2].value > -1}
         {edits[2].value.toString().padStart(4, '0')}
-      {:else}ГГГГ{/if}
+      {:else}YYYY{/if}
```
Comment (BykhovDenis) [inline packages/ui/src/components/calendar/DateRangePresenter.svelte:322]: Probably we need IntlString here

File: plugins/view-resources/src/components/EditDoc.svelte [hunk 2]
```diff
@@ -88,11 +88,15 @@
...
     const descendants = hierarchy.getDescendants(parentClass).map((p) => hierarchy.getClass(p))
+
     mixins = descendants.filter(
       (m) =>
         m.kind === ClassifierKind.MIXIN &&
         !ignoreMixins.has(m._id) &&
-        (hierarchy.hasMixin(object, m._id) || showAllMixins)
+        (hierarchy.hasMixin(object, m._id) ||
+          (showAllMixins &&
+            // hierarchy.isDerived(hierarchy.getBaseClass(m._id), realObjectClass)
```
Comment (BykhovDenis) [inline plugins/view-resources/src/components/EditDoc.svelte:98]: Extra?

**Review by BykhovDenis — APPROVED** [review]

File: packages/ui/src/components/calendar/DateRangePresenter.svelte (hunk 1 above)
Comment (haiodo) [reply packages/ui/src/components/calendar/DateRangePresenter.svelte:322]: Yes, https://front.hc.engineering/workbench/platform/tracker/TSK-465

File: plugins/view-resources/src/components/EditDoc.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/view-resources/src/components/EditDoc.svelte:98]: Heh, will clean on next PR.

**Pull request merged**